reference/latest/src/main/java/com/example/docs/item/ModItems.java

```diff
@@ -39,6 +41,12 @@ public class ModItems {
 	// :::7
 	public static final Item GUIDITE_SWORD = register(new SwordItem(GuiditeMaterial.INSTANCE, new Item.Settings()), "guidite_sword");
 	// :::7
+	// :::13
```

Comment:
Need to be `:::_13` to prevent collisions with `:::1`

Reply:
Fixed :)